getting-started/quickstart.md

```diff
@@ -18,7 +18,7 @@ An application is defined on Clever Cloud by the following elements:
 
 * a dedicated language/framework
 * a deployment method (FTP and/or Git)
-* RAM and CPU consumption, depending on the language or framework used
+* a RAM and CPU consumption, depending on the language or framework used
```

Comment:
I think it was better without the change. But maybe we could enhance it with something like `- Resources consumption (CPU, RAM, Disk, ...), depending on the language or framework used`